cli/consumer_command.go

```diff
@@ -561,11 +561,16 @@ func (c *consumerCmd) leaderStandDownAction(_ *fisk.ParseContext) error {
 }
 
 func (c *consumerCmd) interactiveEdit(cfg api.ConsumerConfig) (*api.ConsumerConfig, error) {
-	editor := os.Getenv("EDITOR")
-	if editor == "" {
+	rawEditor := os.Getenv("EDITOR")
```

Comment:
this seems the same everywhere, we can probably make a helper in `internal/util` something like `EditFile(f string)` and use that in all the following cases, that would be a nice improvement.

But if its too much for you to do that this PR is fine also let us know

Reply:
Can do! Will update the PR shortly.

Reply:
@ripienaar done, and added new tests for it using `echo` as the EDITOR command: https://github.com/nats-io/natscli/pull/1174/commits/0d162c2a8ab1db53711aa34a3cf097f3f164c4e5